jobs/spec/samples_spec.rb

```diff
@@ -67,36 +75,41 @@
 # verify basic_job_sample.rb
   it "basic_job_sample" do
     begin
-      company_created_test = create_company(generate_company());
-      job_generated_test = generate_job(company_created_test.name);
-      job_created_test = create_job(job_generated_test);
-      get_job(job_created_test.name);
-      job_created_test.description = "Updated description";
-      update_job(job_created_test.name, job_created_test);
-      job_created_test.title = "Updated title software Engineer";
-      update_job_with_field_mask(job_created_test.name, "title", job_created_test);
-      delete_job(job_created_test.name);
-      delete_company(company_created_test.name);
+      company_generated_test = job_discovery_generate_company display_name: "Google", 
+                                                              headquarters_address: "1600 Amphitheatre Parkway Mountain View, CA 94043"
+      company_created_test = job_discovery_create_company company_to_be_created: company_generated_test
+      job_generated_test = job_discovery_generate_job company_name: company_created_test.name
+      job_created_test = job_discovery_create_job job_to_be_created: job_generated_test
+      job_discovery_get_job job_name: job_created_test.name
+      job_created_test.description = "Updated description"
+      job_discovery_update_job job_name: job_created_test.name, 
+                               job_to_be_updated: job_created_test
+      job_created_test.title = "Updated title software Engineer"
+      job_discovery_update_job_with_field_mask job_name: job_created_test.name,
+                                               field_mask: "title", 
+                                               job_to_be_updated: job_created_test
+      job_discovery_delete_job job_name: job_created_test.name
+      job_discovery_delete_company company_name: company_created_test.name
       capture = $stdout.string
-      expect(capture).to include("Job created");
-      expect(capture).to include("Job got");
-      expect(capture).to include("Job updated");
-      expect(capture).to include("Job updated with filedMask title");
-      expect(capture).to include("Job deleted");
+      expect(capture).to include("Job created")
+      expect(capture).to include("Job got")
+      expect(capture).to include("Job updated")
+      expect(capture).to include("Job updated with filedMask title")
+      expect(capture).to include("Job deleted")
     rescue
```

Comment:
Capture the rescue error and print it similar as in samples. It will help with future debugging.